packages/compartment-mapper/src/node-modules.js

```diff
@@ -609,6 +704,7 @@ const translateGraph = (
       scopes,
       parsers,
       types,
+      policy: packagePolicy,
```

Comment:
policy specified upon loading is introduced into compartments piece by piece. Thus getting successfully introduced in archives too.